k8s/perftest/cluster-01/static/neuvector/neuvector.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: neuvector
+  namespace: neuvector
+  annotations:
+    flux.weave.works/ignore: "false"
+spec:
+  releaseName: neuvector
+  chart:
+    git: https://github.com/hmcts/neuvector-helm
+    ref: config-map
+    path: "."
+  values:
+    registry: hmctsprivate.azurecr.io
```

Comment:
shouldn't need any of this config if you use the kustomise base? see the aat neuvector file